apps/community/src/components/board/board-list.css.ts

```diff
@@ -0,0 +1,91 @@
+import { screen, themeVars } from '@aics-client/design-system/styles';
+import { globalStyle, style } from '@vanilla-extract/css';
+
+const boardListWrapper = style({
+  display: themeVars.display.flex,
+  flexDirection: themeVars.flexDirection.column,
+  width: themeVars.width.full,
+  borderTop: '1px solid',
+  borderColor: themeVars.color.gray300,
+});
+
+const row = style({
+  display: themeVars.display.flex,
+  alignItems: themeVars.alignItems.center,
+  minWidth: themeVars.width.full,
+  maxHeight: '4rem',
+  gap: '1rem',
+  paddingTop: '1rem',
+  paddingBottom: '1rem',
+  borderBottom: '1px solid',
+  borderColor: themeVars.color.gray300,
+  textOverflow: 'ellipsis',
+  selectors: {
+    '&:hover': {
+      backgroundColor: themeVars.color.gray100,
+    },
+  },
+
+  ...screen.xl({
+    minHeight: '3.5rem',
+  }),
+});
+
+globalStyle(`${row} > * + *`, {
+  textAlign: 'center',
+});
+
+const pin = style({
+  display: themeVars.display.flex,
+  justifyContent: themeVars.justifyContent.center,
+  width: '5%',
+});
+
+const rowTitle = style({
+  display: themeVars.display.flex,
+  justifyContent: themeVars.justifyContent.center,
+  alignItems: themeVars.alignItems.center,
+  width: '90%',
+  gap: '0.5rem',
+  fontWeight: themeVars.fontWeight.semibold,
+
+  ...screen.md({
+    width: '60%',
+    justifyContent: themeVars.justifyContent.start,
+  }),
+});
+
+const information = style({
+  width: '0%',
+  visibility: 'hidden',
+
+  ...screen.md({
+    display: themeVars.display.flex,
+    justifyContent: themeVars.justifyContent.end,
+    alignItems: themeVars.alignItems.center,
+    width: '35%',
+    visibility: 'visible',
+    fontSize: themeVars.fontSize.xs,
+    gap: '0.75rem',
+    paddingRight: '1.5rem',
+  }),
+});
+
+const view = style({
+  display: themeVars.display.flex,
+  justifyContent: themeVars.justifyContent.center,
+  alignItems: themeVars.alignItems.center,
+  gap: '0.25rem',
+});
+
+const author = style({
+  width: '0%',
+  visibility: 'hidden',
+
+  ...screen.xl({
+    visibility: 'visible',
+    width: 'auto',
+  }),
+});
```

Comment:
```suggestion
const author = style([{
  width: '0%',
  visibility: 'hidden'
},
screen.xl({
    visibility: 'visible',
    width: 'auto',
  })
]);
```

배열로 스타일을 주입할 수 있어요 (이런 방법이 있는지 알려주는거에요)